### ISSUE TITLE
Port forward is not working sometimes

### ISSUE THREAD

**Issue by cniackz** [minio/operator#1442]
### Description:

`k port-forward svc/console 9090 -n minio-operator` fails during the login:

```sh
Cesars-MacBook-Pro:~ cniackz$ k port-forward svc/console 9090 -n minio-operator
Forwarding from 127.0.0.1:9090 -> 9090
Forwarding from [::1]:9090 -> 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
E0210 11:21:08.820832   49959 portforward.go:379] error copying from remote stream to local connection: readfrom tcp6 [::1]:9090->[::1]:53087: write tcp6 [::1]:9090->[::1]:53087: write: broken pipe
E0210 11:21:38.806582   49959 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
E0210 11:21:38.821601   49959 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
E0210 11:22:08.823053   49959 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
E0210 11:22:09.208845   49959 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
E0210 11:22:39.590827   49959 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
```

## Expected Behavior:

It should not fail and login should work as it used to work.

## Current Behavior:

Current behavior is in terminal you see:

```sh
Cesars-MacBook-Pro:~ cniackz$ k port-forward svc/console 9090 -n minio-operator
Forwarding from 127.0.0.1:9090 -> 9090
Forwarding from [::1]:9090 -> 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
E0210 11:21:08.820832   49959 portforward.go:379] error copying from remote stream to local connection: readfrom tcp6 [::1]:9090->[::1]:53087: write tcp6 [::1]:9090->[::1]:53087: write: broken pipe
E0210 11:21:38.806582   49959 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
E0210 11:21:38.821601   49959 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
E0210 11:22:08.823053   49959 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
E0210 11:22:09.208845   49959 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
E0210 11:22:39.590827   49959 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
```

or 

```sh
^CCesars-MacBook-Pro:~ cniackz$ k port-forward svc/console 9090 -n minio-operator
Forwarding from 127.0.0.1:9090 -> 9090
Forwarding from [::1]:9090 -> 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
Handling connection for 9090
E0210 11:25:02.162807   50413 portforward.go:379] error copying from remote stream to local connection: readfrom tcp6 [::1]:9090->[::1]:53380: write tcp6 [::1]:9090->[::1]:53380: write: broken pipe
Handling connection for 9090
E0210 11:25:32.139587   50413 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
E0210 11:25:32.139823   50413 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
Handling connection for 9090
E0210 11:25:32.152836   50413 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
E0210 11:25:32.164152   50413 portforward.go:345] error creating error stream for port 9090 -> 9090: Timeout occurred
Handling connection for 9090
```

* And in UI you see that it get stucks forever:

<img width="1840" alt="Screenshot 2023-02-10 at 11 24 39 AM" src="https://user-images.githubusercontent.com/6667358/218143260-f3f0d140-cd22-4ee2-873d-96fbbe3eac31.png">

<img width="1840" alt="Screenshot 2023-02-10 at 11 25 38 AM" src="https://user-images.githubusercontent.com/6667358/218143500-59574958-0c49-4a6e-aa9e-8ea7dfb9a567.png">

![image](https://user-images.githubusercontent.com/6667358/218143534-6302903f-f160-47ee-83c8-3230e27f9f47.png)

## Possible Solution

There is no current solution provided.

But some users report:

1. Use Node Port will solve it
2. Restart the port forward and Delete `login` from URL that might fix it in some cases, not always!, not for everyone!

## Steps to Reproduce (for bugs)

1. Create a kind cluster
2. Install MinIO Operator
3. Port forward the service
4. Login via Browser

I added more documentation a month ago: https://github.com/cniackz/public/wiki/Operator-Login-Fail
And as today Friday February 10th 2023 this is still an issue.

## Context

This is affecting many users already in testing clusters, the problem started in version 4.5.6 of Operator, as a fun fact, in version 4.5.5 it worked well, so something with the commit that created tag 4.5.6 is introducing the issue. It would be nice to root cause what line introduced the issue but there are several changes in that commit so it might be difficult to root cause.

## Regression

Yes this was found during 4.5.6 regression and in 4.5.5 was working.

## Your Environment

* Version used (`minio-operator`): 4.5.6 onwards 
* Environment name and version (e.g. kubernetes v1.17.2): 

```
^CCesars-MacBook-Pro:~ cniackz$ kubectl version
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"26", GitVersion:"v1.26.0", GitCommit:"b46a3f887ca979b1a5d14fd39cb1af43e7e5d12d", GitTreeState:"clean", BuildDate:"2022-12-08T19:51:43Z", GoVersion:"go1.19.4", Compiler:"gc", Platform:"darwin/arm64"}
Kustomize Version: v4.5.7
Server Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.3", GitCommit:"434bfd82814af038ad94d62ebe59b133fcb50506", GitTreeState:"clean", BuildDate:"2022-10-25T19:38:29Z", GoVersion:"go1.19.2", Compiler:"gc", Platform:"linux/arm64"}
```

* Server type and version:

Using Kind on:
Macbook Pro M1 Pro

* Operating System and version (`uname -a`):

```sh
Cesars-MacBook-Pro:~ cniackz$ uname -a
Darwin Cesars-MacBook-Pro.local 22.3.0 Darwin Kernel Version 22.3.0: Thu Jan  5 20:48:54 PST 2023; root:xnu-8792.81.2~2/RELEASE_ARM64_T6000 arm64
```

* Link to your deployment file:

Not apply.

### Root causing:

As per internal conversation with Harsha:

```
That is because console version is different
```

As per internal conversation with Ravind:

```
I believe 0.22.1 added the websocket behavior where things got a lot trickier
```

* Maybe this should be a console issue not Operator one!

### Console issue:

https://github.com/minio/console/issues/2539

**Comment by dvaldivia** [comment]
the root cause is the `net/http` package doing something when we start serving a video asset for the background, I'm still looking into this

**Comment by AnttiKarvinen** [comment]
Great that someone is looking into this, our test-environment uses a lot of Minio and it would be good to get onboard with our datasection.

**Comment by xoxys** [comment]
Wouldn't it make sense to prioritize usability and remove the video for now until the problem is fixed?

**Comment by dvaldivia** [comment]
this has been fixed in `v5.0.2`
